kratos/python_scripts/kratos_utilities.py

```diff
@@ -61,7 +61,7 @@ def CheckIfApplicationsAvailable(*application_names):
     available_apps = GetListOfAvailableApplications()
     return all(app_name in available_apps for app_name in application_names)
 
-def GetNotAvailableApplications(*application_names):
```

Comment:
if you do this then you can no longer call it with a variable amount of argument

Reply:
I changed it because, no where else we are using it, but I get your point :)